client/packages/system/src/Asset/ListView/ListView.tsx

```diff
@@ -109,25 +161,17 @@ const AssetListComponent: FC = () => {
         importModalController={importModalController}
         assets={data?.nodes ?? []}
       />
-      <Toolbar />
-      <DataTable
-        id="item-list"
-        pagination={{ ...pagination, total: data?.totalCount ?? 0 }}
-        onChangePage={updatePaginationQuery}
-        columns={columns}
-        data={data?.nodes}
-        isError={isError}
-        isLoading={isLoading}
-        noDataElement={<NothingHere body={t('error.no-assets')} />}
-        enableColumnSelection
+
+      <MaterialTable table={table} />
+
+      <Footer
+        selectedRows={selectedRows}
+        resetRowSelection={table.resetRowSelection}
       />
-      <Footer />
     </>
   );
 };
 
 export const AssetListView: FC = () => (
-  <TableProvider createStore={createTableStore}>
-    <AssetListComponent />
-  </TableProvider>
+  <AssetListComponent />
```

Comment:
Just get rid of this wrapper completely now -- call the main component `AssetListView` instead of `AssetListComponent` and export it directly.